rpm/hosted-ce-tools.spec

```diff
@@ -10,6 +10,11 @@ BuildArch: noarch
 Requires: python-six
 Requires: fetch-crl
 Requires: sudo
+%if 0%{?rhel} < 8
+Requires: git
+%else
+Requires: git-core
```

Comment:
I take it that on el8, `git-core` gives a more minimal git install, that is still sufficient for the purposes of this script?

Alternatively you could just require `/usr/bin/git`, which installs `git` or `git-core` as needed.

Reply:
I like that.